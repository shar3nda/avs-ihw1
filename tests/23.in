10
5 2 2 6 0 1 -2 2 6 -1