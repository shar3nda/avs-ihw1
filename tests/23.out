-6 -9 4 0 -2 3 5 5 -2